### PR TITLE
Remove compilation warning

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
@gkoz: Ping !

### CODE REVIEW

File: src/object.rs [hunk 1]
```diff
@@ -171,7 +171,8 @@ pub trait Downcast<T> {
     /// Tries to downcast to `T`.
     ///
     /// Returns `Ok(T)` if the instance implements `T` and `Err(Self)` otherwise.
-    fn downcast(self) -> Result<T, Self>;
+    fn downcast(self) -> Result<T, Self>
+       where Self: Sized;
```
Comment (gkoz) [inline src/object.rs]: Odd indent. Otherwise :+1:

Comment (GuillaumeGomez) [inline src/object.rs]: Arf, emacs indent. I change it !

**Comment by GuillaumeGomez** [comment]
@gkoz: Updated.

**Comment by gkoz** [comment]
Thanks.

**Pull request merged**